pkgs/top-level/all-packages.nix

```diff
@@ -10897,6 +10897,7 @@ let
   mutt-with-sidebar = callPackage ../applications/networking/mailreaders/mutt {
     withSidebar = true;
   };
+  mutt-kz = callPackage ../applications/networking/mailreaders/mutt/mutt-kz.nix { };
```

Comment:
This seems a different software, can you please put it in a different directory?